packages/@webex/plugin-meetings/test/unit/spec/breakouts/index.ts

```diff
@@ -398,7 +398,9 @@ describe('plugin-meetings', () => {
           breakoutMoveId: 'breakoutMoveId',
         });
 
-        assert.calledOnce(breakoutEvent.onBreakoutJoinResponse);
+        assert.calledOnce(onBreakoutJoinResponseSpy);
+
+        onBreakoutJoinResponseSpy.restore()
```

Comment:
Is there a reason we need to restore this if we are creating the spy in the next test again?

Reply:
Yes, we need to restore spies because if we dont restore the spied and try to stub the same method again. It gives the error 'Trying to wrap a function which is already a stub'.